src/rcx/include/rcx/ext2dBox.h

```diff
@@ -0,0 +1,86 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (c) 2023, Nefelus Inc
```

Comment:
This should be "// Copyright (c) 2019-2023, Nefelus Inc, Google LLC"

Reply:
Updated copyright notices on touched files, PTAL.

Reply:
LGTM